dts/bindings/clock/renesas,rx-cgc-root-clock.yaml

```diff
@@ -43,7 +43,13 @@ properties:
     enum:
       - 0
       - 1
+      - 2
+      - 3
+      - 4
     description: |
       Drive Capacity Control (for Sub-Clock Oscillator only)
       0: Drive capacity for standard CL.
-      1: Drive capacity for low CL.
+      1: Drive capacity for low CL. (in case of rx261, not choose this)
```

Comment:
```suggestion
      1: Drive capacity for low CL. (in case of rx261, do not choose this)
```

or

```suggestion
      1: Drive capacity for low CL. (unsupported on rx261)
```

?